api/src/opentrons/protocol_engine/state/modules.py

```diff
@@ -681,9 +681,9 @@ def should_dodge_thermocycler(
         Returns True if we need to dodge, False otherwise.
         """
         all_mods = self.get_all()
-        if all_mods and ModuleModel.THERMOCYCLER_MODULE_V1 in [
-            mod.model for mod in all_mods
-        ]:
+        if any(
+            [ModuleModel.is_thermocycler_module_model(mod.model) for mod in all_mods]
```

Comment:
IIRC the brackets are not needed, because `any` may take a generator expression

```suggestion
            ModuleModel.is_thermocycler_module_model(mod.model) for mod in all_mods
```